h2/src/test/org/h2/test/db/TestGeneralCommonTableQueries.java

```diff
@@ -42,6 +43,8 @@ public void test() throws Exception {
         testMerge();
         testCreateTable();
         testNestedSQL();
+        //testRecursiveTable();
```

Comment:
???

Reply:
There were always going to be conflicts until the other pull request was integrated - I resolved the conflicts and fixed the commented out test... Lets see if it passes !